lib/rubocop/cop/style/one_line_conditional.rb

```diff
@@ -5,6 +5,15 @@ module Cop
     module Style
       # TODO: Make configurable.
       # Checks for uses of if/then/else/end on a single line.
+      #
+      # @example
+      #   # bad
+      #   if foo then boo else doo end
+      #   if foo then boo end
+      #   unless foo then boo end
+      #
+      #   # good
+      #   foo ? boo : doo
```

Comment:
I think here you should also mention that the multi-line versions of those expressions would be fine. Or the suffix forms for the examples without else.

Reply:
Actually I think the cop doesn't even check expressions without an `else`.

Reply:
@bbatsov you are right >_<. I'm so used to not write oneliners with "if/then/end" style, that I missed that examples in specs :(

Fixed.